headers-ext/src/lib.rs

```diff
@@ -0,0 +1,25 @@
+extern crate base64;
+extern crate bytes;
+extern crate headers_core;
+#[macro_use]
+extern crate headers_derive;
+extern crate http;
+extern crate mime;
+extern crate sha1;
+extern crate time;
+
+use headers_core::{
```

Comment:
What do you think about `pub use headers_core::HeaderMapExt` here ? It would allow to use the headers_ext crate only